src/brucke_app.erl

```diff
@@ -37,7 +37,7 @@ graphite_host() -> app_env(graphite_host).
 
 graphite_port() -> app_env(graphite_port).
 
-http_port() -> app_env(http_port, 8080).
+http_port() -> app_env(healthcheck_port, 8080).
```

Comment:
```suggestion
app_env(healthcheck_port, app_env(healthcheck_port, 8080)).
```
to be backward compatible.

Reply:
Thank you vary much! Good suggestion! I'm not thoughtful enough.